capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -40,4 +30,31 @@ const BookSearchTile = (props) => {
   );
 }
 
+const BookListAddDropdown = (props) => {
```

Comment:
(No action required) nit: Consider destructuring `props` here, i.e. `({ book, userBookLists })`. This makes the function signature more informative and allows you to omit `props.` in the JSX below.

Reply:
This is a good idea! I didn't know about it before